conf/spark-defaults.conf.template

```diff
@@ -25,3 +25,4 @@
 # spark.serializer                 org.apache.spark.serializer.KryoSerializer
 # spark.driver.memory              5g
 # spark.executor.extraJavaOptions  -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
+# spark.sql.catalogImplementation  hive
```

Comment:
I also wouldn't document this here. This config is largely meant to be for internal use.